packages/shared/pkg/db/snapshot.go

```diff
@@ -182,20 +182,17 @@ func (db *DB) GetSnapshotBuilds(ctx context.Context, sandboxID string, teamID uu
 			env.HasSnapshotsWith(snapshot.SandboxID(sandboxID)),
 			env.TeamID(teamID),
 		).
-		WithSnapshots(func(query *models.SnapshotQuery) {
-			query.Where(snapshot.SandboxID(sandboxID)).Only(ctx)
-		}).
 		WithBuilds().
 		Only(ctx)
 
 	notFound := models.IsNotFound(err)
 
-	if err != nil {
-		return nil, nil, fmt.Errorf("failed to get snapshot build for '%s': %w", sandboxID, err)
+	if notFound {
+		return nil, nil, EnvNotFound{}
 	}
 
-	if notFound {
-		return nil, nil, fmt.Errorf("no snapshot build found for '%s'", sandboxID)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to get snapshot build for '%s': %w", teamID, err)
```

Comment:
why is team id here?

Reply:
it's second time it's happening this week, it overwrites my error messages for some reason